packages/plugin-core/src/windowWatcher.ts

```diff
@@ -11,6 +15,12 @@ import { debouncedUpdateDecorations } from "./features/windowDecorations";
 import { Logger } from "./logger";
 import { AnalyticsUtils, sentryReportingCallback } from "./utils/analytics";
 
+const trackScrolled = _.debounce(() => {
```

Comment:
debounce helps. want to see if users have used the tutorial (since we're dropping them in, there's no indicator from the green button anymore). limited this to just tutorial notes